training/src/hydra_plugins/anemoi_searchpath/anemoi_searchpath_plugin.py

```diff
@@ -68,4 +68,14 @@ def manipulate_search_path(self, search_path: ConfigSearchPath) -> None:
                 )
                 LOGGER.info("Prepending current user directory (%s) to the search path.", cwd_path)
                 LOGGER.debug("Search path is now: %s", search_path)
+
+        # Add package config path as lowest priority fallback (issue #570)
+        # This enables discovery of default configs like 'training/default'
```

Comment:
thanks Jesper for looking into this! this shouldn't be conflicting with the anemoi-training train command as in it won't append the path there twice right?